charts/cluster-api-cluster-openstack/files/patches/kube-apiserver+json.tpl

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "op": "replace",
+        "path": "/spec/containers/0/resources/requests/cpu",
```

Comment:
You are aware this isn't an enforced limit at all, it's more of a burstable scheduling constraint, allowing over commit.  I'm guessing making it a limit isn't what you want either due to OOM killing...

Reply:
Yeah brainfart here (should be applied to limits not requests), I _do_ in fact want to enforce a hard limit on kube-apiserver if it runs away with memory, lest the whole machine run out of memory and cause even worse chaos.